components/content_settings/core/browser/brave_content_settings_pref_provider.cc

```diff
@@ -433,28 +443,23 @@ bool BravePrefProvider::SetWebsiteSettingInternal(
 }
 
 std::unique_ptr<RuleIterator> BravePrefProvider::GetRuleIterator(
-      ContentSettingsType content_type,
-      bool incognito) const {
+    ContentSettingsType content_type,
+    bool incognito) const NO_THREAD_SAFETY_ANALYSIS {
```

Comment:
this work only for locks on-stack. using something like `std::unique_ptr<base::AutoLock>` doesn't work with analysis.